vllm/entrypoints/openai/cli_args.py

```diff
@@ -169,6 +169,8 @@ class FrontendArgs:
     log. The default of None means unlimited."""
     disable_fastapi_docs: bool = False
     """Disable FastAPI's OpenAPI schema, Swagger UI, and ReDoc endpoint."""
+    disable_sagemaker_standards: bool = False
```

Comment:
given you added this as default dependencies, there's no need for an extra flag to turn it off?